lib/fibonacci.rb

```diff
@@ -7,6 +7,37 @@
 
 # Time complexity: ?
 # Space complexity: ?
+
+## this works but feels bad
```

Comment:
Don't feel bad about this solution.  It's pretty good. It actually uses less memory.